packages/@aws-cdk/custom-resources/lib/aws-custom-resource/runtime/index.ts

```diff
@@ -39,6 +44,20 @@ function decodeBooleans(object: object) {
   });
 }
 
+/**
+ * Decodes physical resource id
+ */
+function decodePhysicalResourceId(object: object, physicalResourceId: string) {
```

Comment:
I think I'd rather you fold this into `decodeBooleans` (and probably rename it to `decodeSpecialValues` or something).